src/templates/snek/gql/tasks/index.ts

```diff
@@ -0,0 +1,82 @@
+//#region > Imports
+//#PACKAGE "graphql"
+//## npm install "graphql"@14.6.0
+// Contains the interface for gql queries, mutations and subscriptions
+import { DocumentNode } from "graphql";
+
+//> Sessions
+// Contains the SNEK session
+import { SnekSession } from "../../../../session/sessions";
+//> Tasks
+// Contains the SnekGqlAuth task
+import SnekGqlAuthTasks from "./auth";
+// Contains the SnekGqlGeneral task
+import SnekGqlGeneralTasks from "./general";
+// Contains the SnekGqlUser task
+import SnekGqlUserTasks from "./user";
+// Contains error handling for tasks
+import { TaskError } from "../errors";
+//> Interfaces
+// Contains the snek template
+import { SnekGqlTemplate } from "../index";
+//> Types
+// Contains the type declaration for apollo results
+import { ApolloResult } from "../index";
+//#endregion
+
+//#region > Classes
+/** @class A Template with initializes all tasks */
+class SnekTasks extends TaskError {
+  public template!: SnekGqlTemplate;
+  public general!: SnekGqlGeneralTasks;
+  public auth!: SnekGqlAuthTasks;
+  public user!: SnekGqlUserTasks;
+
+  /**
+   * @constructor
+   * @author Nico Schett <contact@schett.net>
+   * @param {string} session A session to initialize all corresponding tasks
+   */
```

Comment:
Should have some sort of comment.

?

Reply:
Yeah, but not your suggestion cause this includes too less/useless information.
We have to discuss about a new TONIC rule in snek-at/tonic/issues/3